### PR TITLE
Fix bug where we can't see counts in Insight Table

### LINKED ISSUE #2983: Show Cohort name instead of number on table views
## Is your feature request related to a problem?

*Please describe.*

I have no idea what these cohort numbers mean when looking at a table.

## Describe the solution you'd like

Use cohort name instead of number on tables (with a cut-off length ofc). This could be instead of or perhaps even in addition to the cohort number.

Screenshot 2021-01-18 at 13 19 26

## Describe alternatives you've considered

Leave as is.

### PR DESCRIPTION
**Pull request by alexkim205**
## Changes

Fixes #2983

## Checklist

- [ ] All querysets/queries filter by Organization, by Team, and by User
- [ ] Django backend tests
- [ ] Jest frontend tests
- [ ] Cypress end-to-end tests
- [ ] Migrations are safe to run at scale (e.g. PostHog Cloud) – present proof if not obvious
- [ ] Frontend/CSS is usable at 320px (iPhone SE) and decent at 360px (most phones)

### CODE REVIEW

**Comment by paolodamico** [comment]
Think it's worth considering already changes from #4700

**Comment by alexkim205** [comment]
> Think it's worth considering already changes from #4700

That PR is so nice, love the UI changes. I think my fix in this PR would still apply because I'm not editing the InsightsTable component itself, rather just passing the filter into the `<BindLogic/>` which wraps both `<InsightsTableV1/>` and `<InsightsTableV2/>`.

I'm also going to generalize the cypress tests in my PR so that we don't have to touch them again once we move from V1 to V2.

**Review by paolodamico — APPROVED** [review]

File: frontend/src/scenes/insights/Insights.tsx [hunk 1]
```diff
@@ -374,9 +374,14 @@ export function Insights(): JSX.Element {
                                     <Card style={{ marginTop: 16 }}>
                                         <BindLogic
                                             logic={trendsLogic}
-                                            props={{ dashboardItemId: null, view: activeView }}
+                                            props={{ dashboardItemId: null, view: activeView, filters: allFilters }}
                                         >
-                                            <InsightsTable />
+                                            <InsightsTable
+                                                showTotalCount={
+                                                    featureFlags[FEATURE_FLAGS.NEW_TOOLTIPS] &&
```
Comment (mariusandra) [inline frontend/src/scenes/insights/Insights.tsx:381]: Is this bugfix something we want to feature flag? 🤔

Comment (paolodamico) [reply frontend/src/scenes/insights/Insights.tsx:381]: We're not feature flagging the bug fix, `showTotalCount` is used to determine if we should display the new total count column in the table view, so I think it's fine.

**Review by mariusandra — CHANGES_REQUESTED** [review]
I'm probably doing something dumb here, but with [this query](http://localhost:8000/insights?insight=TRENDS&interval=day&events=%5B%7B%22id%22%3A%22%24pageview%22%2C%22name%22%3A%22%24pageview%22%2C%22type%22%3A%22events%22%2C%22order%22%3A0%7D%5D&properties=%5B%5D&filter_test_accounts=false&breakdown=false&breakdown_type&compare=false&date_from=-7d&display=ActionsTable)

a) if I have no feature flags enabled, I see just a broken table
b) if I have the new tooltips flag enabled, I see this:

image

even if there is actually some data:

image

File: frontend/src/scenes/insights/Insights.tsx (hunk 1 above)
Comment (mariusandra) [reply frontend/src/scenes/insights/Insights.tsx:381]: Yeah, but we just show it to those who are using the new experimental UI, even though the bug is there for everyone

**Comment by alexkim205** [comment]
@mariusandra 

a. The bug exists in both the old and new ui, so hiding this behind a feature flag doesn't make sense. I'll remove that check.
b. I clicked the link with the feature flag enabled but can't reproduce the issue. I tried to use some data that had similar shape (where data only exists on last day) [here](http://localhost:8000/insights?insight=TRENDS&interval=day&display=ActionsLineGraph&events=%5B%7B%22id%22%3A%22%24pageview%22%2C%22name%22%3A%22%24pageview%22%2C%22type%22%3A%22events%22%2C%22order%22%3A0%7D%5D&properties=%5B%5D&filter_test_accounts=false&date_from=2021-06-19&date_to=2021-06-21) but it looks like all the data's there 🤔

**Review by mariusandra — APPROVED** [review]

**Pull request merged**